lib/syntax_tools/test/syntax_tools_SUITE_data/syntax_tools_test.erl

```diff
@@ -48,8 +49,7 @@ foo1() ->
 %% macro test
 foo2(A,B) ->
     % string combining ?
-    [?macro_string, ?macro_string
-     ?macro_string,
+    [?macro_string, ?macro_string ++
```

Comment:
I added a check in the test suite that `epp_dodger` does not return any error marks for better verification that it does its job. This is the only place where it did. Here `epp_dodger` has no chance of realizing that the 2 consecutive macros are two strings so I added an explicit concatenation. Additionally the `epp_dodger_clever` testcase contains scenarios when `epp_dodger` actually has a chance (one of the two positions is a literal string)